### PR TITLE
messages added for empty fields on course and course run page.

### PR DESCRIPTION
**Pull request by tasawernawaz**
[ECOM-7234](https://openedx.atlassian.net/browse/ECOM-7234)

* All empty or null fields are listed with the string "(Optional) Not yet added" for optional fields for both course and course run page.
* All empty or null fields are listed with the string "(Required) Not yet added" for required fields for both course and course run page.

**Course Page:**
screen shot 2017-03-01 at 4 06 08 pm

**Course Run Page:**
screen shot 2017-03-01 at 4 06 22 pm
screen shot 2017-03-01 at 4 06 30 pm
screen shot 2017-03-01 at 4 06 37 pm
screen shot 2017-03-01 at 4 06 45 pm
screen shot 2017-03-01 at 4 06 55 pm

### CODE REVIEW

**Comment by tasawernawaz** [comment]
@awais786 @waheedahmed Please review this.

File: course_discovery/templates/publisher/course_run_detail/_all.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
       {% if object.studio_url %}
         <a target="_blank" href="{{ object.studio_url }}">{{ object.lms_course_id }}</a>
       {% else %}
-        {% trans "empty" %}
+        {% trans "(Optional) Not yet added" %}
```
Comment (waheedahmed) [inline course_discovery/templates/publisher/course_run_detail/_all.html]: Can you please confirm this with Deen that `lms_course_id` is required field or not?

**Review by waheedahmed — APPROVED** [review]
Just one question

**Review by awais786 — APPROVED** [review]

File: course_discovery/templates/publisher/course_run_detail/_all.html (hunk 1 above)
Comment (bderusha) [reply course_discovery/templates/publisher/course_run_detail/_all.html]: Required before it can be submitted for review

Comment (tasawernawaz) [reply course_discovery/templates/publisher/course_run_detail/_all.html]: There are many test cases are failing due to this change, so I am creating a new ticket to make `lms_course_id` required and fix all related test cases. [ECOM-7358](https://openedx.atlassian.net/browse/ECOM-7358)

**Pull request merged**